MantidQt/API/src/WorkspaceObserver.cpp

```diff
@@ -4,6 +4,10 @@
 #include "MantidQtAPI/WorkspaceObserver.h"
 #include "MantidAPI/AnalysisDataService.h"
 
+//#include <QMetaType>
```

Comment:
Minor point, but better not to have dead code. Maybe fix if I find anything more serious.